src/core/include/openvino/opsets/opset10_tbl.hpp

```diff
@@ -196,3 +196,6 @@ _OPENVINO_OP_REG(ROIAlign, ov::op::v9)
 _OPENVINO_OP_REG(SoftSign, ov::op::v9)
 _OPENVINO_OP_REG(GenerateProposals, ov::op::v9)
 _OPENVINO_OP_REG(MulticlassNms, ov::op::v9)
+
+// New operations added in opset10
+_OPENVINO_OP_REG(IsInf, ov::op::v10)
```

Comment:
Please add `IsInf` to `OpImplCheckTest` 
Example for unary operations:
https://github.com/openvinotoolkit/openvino/blob/78d6f68de4f3b4d358b8e8fc0b066758e500f9cc/src/tests/functional/plugin/shared/src/single_layer_tests/op_impl_check/single_op_graph.cpp#L1282-L1287

Reply:
Done